README.md

```diff
@@ -66,6 +66,7 @@ The SDK supports full scale of the API querying and filtering capabilities as de
 // the 'brewer' content type, ordered by the 'product_name' element value
 ContentItemsListingResponse response = client.getItems(
     DeliveryParameterBuilder.params()
+        .language("es-ES")
```

Comment:
We should put in a Github issue to allow this to take a Java Locale, as that's likely what a consumer will get from other libs.  It's format is "es_ES".

Some examples to create them:
```
Locale.UK.toString()
new Locale("en", "GB").toString()
```